src/ext/ca/wixca/dll/secureobj.cpp

```diff
@@ -690,7 +703,7 @@ extern "C" UINT __stdcall ExecSecureObjects(
         //
         ea.grfAccessMode = SET_ACCESS;
 
-        if (0 == lstrcmpW(L"CreateFolder", pwzTable))
+        if (dwAttributes & SECURE_OBJECT_ATTRIBUTE_INHERITABLE)
```

Comment:
@robmen Does this need to be added, since there is special-case code in UtilCompiler.ParsePermissionExElement() that will make Inheritable the default for CreateFolder elements?